src/app/admin/pontuacao/edit-pontuacao/edit-pontuacao.component.ts

```diff
@@ -44,27 +52,70 @@ export class EditPontuacaoComponent implements OnInit {
 
   editPontuacao(event) {
     const formval = this.formPontuacao.value;
+    this.querying = true;
+    this.spinner.show();
     this.pontuacaoservice.patchPontuacao(this.pontuacaoselecionada.id, formval).subscribe(succ => {
+      this.querying = false;
+      this.SpinnerTimeout();
+      succ['nomealuno'] = succ['aluno'].fullName;
+      succ['anoaluno'] = succ['aluno'].ano;
+      succ['atividadetitulo'] = succ['atividade'].titulo;
```

Comment:
Você pode fazer algo como
```suggestion
      this.update['atividadetitulo'] = succ['atividade'].titulo;
```
nessa linha e nas de cima ao invés de editar o objeto que vem do subscribe